xmppserver/src/main/java/org/jivesoftware/openfire/spi/MINAConnectionAcceptor.java

```diff
@@ -1,15 +1,9 @@
 package org.jivesoftware.openfire.spi;
```

Comment:
Given that this class appears to be MINA-specific, shouldn't it be dropped completely?